projects/baby_driver/inc/babydriver_msg_defs.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+// Definitions of the IDs that can go in the first slot of a babydriver CAN message.
+
+typedef enum {
+  // The status message, sent whenever an operation is finished to indicate its status.
+  // Message data: uint8 status.
+  BABYDRIVER_MESSAGE_STATUS = 0,
+
+  NUM_BABYDRIVER_MESSAGES,
+} BabydriverMessageId;
```

Comment:
we could codegen this file with protobufs? I would love it if you somehow got protobufs-c into our infra :))) otherwise I don’t actually know how our C codegen works lol

Reply:
it would be super cool to codegen the babydriver messages for python and C! probably something for another ticket though